config/formats.js

```diff
@@ -423,24 +423,24 @@ let Formats = [
 			'Illegal', 'Unreleased', 'Shedinja', 'Infiltrator', 'Magic Guard', 'Misty Surge', 'Assault Vest', 'Choice Scarf', 'Explosion',
 			'Final Gambit', 'Healing Wish', 'Lunar Dance', 'Magic Room', 'Memento', 'Misty Terrain', 'Self-Destruct',
 		],
-		onValidateTeam: function (team) {
+		getEvoFamily: function (species) {
```

Comment:
The build is failing because this isn’t a possible property on FormatsData in the globals.d.ts file